tests/cluecode/data/plugin_fingerprint/fingerprints.expected.json

```diff
@@ -0,0 +1,39 @@
+{
+  "headers": [
+    {
+      "tool_name": "scancode-toolkit",
+      "options": {
+        "input": "<path>",
+        "--fingerprint": true,
+        "--json": "<file>",
+        "--strip-root": true
+      },
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "message": null,
+      "errors": [],
+      "extra_data": {
+        "files_count": 3
+      }
+    }
+  ],
+  "files": [
+    {
+      "path": "factorial-test.java",
+      "type": "file",
+      "fingerprint": "00000001110011111011010000110111101011001110110100111111011111011000111001000001100001001111010011011000011001110011000100100111",
```

Comment:
is there another way to store this fingerprint, or does it have to be a binary string? The length of these fingerprints is unsightly IMO.

Reply:
@arnav-mandal1234 Store the fingerprint as a hex string instead of a binary string

Reply:
@majurg @JonoYang We can keep the fingerprint in hex but then we have to shift most of the code to the delta code repo and do the computation there. Would it be the right thing to do it?

Reply:
We just want to calcuate the fingerprints for each file in `scancode-toolkit`. When it comes to the actual comparison aspect, that will be handled in `deltacode` (calculating the hamming distance, etc). I do not think we have to move any code over, correct? The additions you made were simply to:

Reply:
@majurg Yes. That is what in the back of my mind too. 
Sorry I misunderstood @JonoYang 's comment. So, basically you want these fingerprints to be in hex. then after we get these hex string on the deltacode side, we again convert them to binary and calculate hamming distance. Right?

Reply:
Yes